components/resolved_ts/src/advance.rs

```diff
@@ -44,6 +46,8 @@ use txn_types::TimeStamp;
 use crate::{endpoint::Task, metrics::*};
 
 const DEFAULT_CHECK_LEADER_TIMEOUT_DURATION: Duration = Duration::from_secs(5); // 5s
+const DEFAULT_GRPC_GZIP_COMPRESSION_LEVEL: usize = 2;
```

Comment:
I'm just curious how is it determined? From the graph seems there's no obvious difference among levels. 
BTW did you observe any impact on CPU utilization?

Reply:
I copy from the default level from another config.

https://github.com/tikv/tikv/blob/9aad050bc7377a1745568782ab7dd8278ccf0e3a/src/server/config.rs#L29

Will attach the test result with CPU utilization later.

Reply:
This change consumes more CPU, the test result is attached in the description, I think it's acceptable.

Reply:
Perhaps we could create some performance regression tests for this pull request using the benchbot, particularly ones that have limited cluster resources.